letta/llm_api/openai.py

```diff
@@ -290,6 +299,7 @@ def openai_chat_completions_process_stream(
                     else:
                         accum_message.content += content_delta
 
+                # TODO(charles) make sure this works for parallel tool calling?
```

Comment:
note to future self (& @sarahwooders  @mattzh72 ) - when we enable parallel tool calling, I have a feeling this code will break (not totally sure)